plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusSessionProperties.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.prometheus;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import io.airlift.units.Duration;
+import io.trino.plugin.base.session.PropertyMetadataUtil;
+import io.trino.plugin.base.session.SessionPropertiesProvider;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.session.PropertyMetadata;
+
+import java.util.List;
+
+public final class PrometheusSessionProperties
+        implements SessionPropertiesProvider
+{
+    private static final String PROMETHEUS_QUERY_CHUNK_SIZE_DURATION = "prometheus_query_chunk_size_duration";
+    private static final String PROMETHEUS_MAX_QUERY_RANGE_DURATION = "prometheus_max_query_range_duration";
```

Comment:
```suggestion
    private static final String QUERY_CHUNK_SIZE_DURATION = "query_chunk_size_duration";
    private static final String MAX_QUERY_RANGE_DURATION = "max_query_range_duration";
```

Reply:
Please document these session properties in prometheus.md.

Reply:
Done